tests/dependency_analyzer/expected_results/skipdep_apache_maven.dl

```diff
@@ -0,0 +1,10 @@
+/* Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved. */
+/* Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/. */
+
+#include "prelude.dl"
+
+Policy("check-dependencies", component_id, "Check the dependencies of pkg:github.com/apache/maven@3fc399318edef0d5ba593723a24fff64291d6f9b") :-
+    .
```

Comment:
Hmm why is this policy empty? Is this expected?

Reply:
It seems the file this policy was created from was empty. Might have to manually create a policy for that case.

Reply:
This looks to be converted from a dependency result file. Can you elaborate on how you convert a dependency result file to a policy file? Thanks.

Reply:
The dependency result files were bulk converted using a script. It takes entries in the files and turns them into lines within the final policy. In this case the file was an empty list so no policy lines were created.